scripts/utils/validationUtils.js

```diff
@@ -0,0 +1,85 @@
+import { EXIST_EMAIL, EXIST_PASSWORD } from './constant.js';
+import { showError, removeError } from './errorUtils.js';
+import { $confirmPasswordInput, $emailInput, $passwordInput } from './tags.js';
+
+const isValidEmail = () => {
+  const emailRegex = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+  return emailRegex.test($emailInput.value);
+};
+const isValidPassword = () => {
+  const passwordRegex = /^(?=.*[a-zA-Z])(?=.*[0-9]).{8,}$/;
+  return passwordRegex.test($passwordInput.value);
+};
+const isMatchPassword = () => $passwordInput.value === $confirmPasswordInput.value;
+const isEmailExisting = () => $emailInput.value === EXIST_EMAIL;
+const isPasswordExisting = () => $passwordInput.value === EXIST_PASSWORD;
+
+const checkValidEmail = () => {
+  if (isValidEmail()) {
+    removeError($emailInput);
+    return;
+  }
+
+  const message = $emailInput.value ? '올바른 이메일 주소가 아닙니다.' : '이메일을 입력해주세요.';
+  showError($emailInput, message);
+};
+
+const checkExistEmail = () => {
+  if (!isEmailExisting()) return;

```

Comment:
개인적으로 return만 해도 {}를 사용하는 것을 지향해요.
전체적인 코드 통일성도 가지고 갈 수 있고, 가독성을 높이는 효과가 있더라구요!

```suggestion
  if (!isEmailExisting()) {
    return;
  }

```